src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp

```diff
@@ -46,25 +47,63 @@ bool EliminateFakeQuantizeTransformation::transform(TransformationContext& conte
 }
 
 namespace {
-bool check_interval(const std::shared_ptr<opset1::Constant>& constant, const float value) noexcept {
+bool check_interval(const std::shared_ptr<opset1::FakeQuantize>& fq,
+                    const std::shared_ptr<opset1::Constant>& constant,
+                    const float value,
+                    const float max_diff,
+                    const bool exact_comparison) noexcept {
+    bool need_to_check_intervals = false;
     const auto& constant_values = constant->cast_vector<float>();
     for (const auto constant_value : constant_values) {
         if (std::fabs(constant_value - value) > std::numeric_limits<float>::epsilon()) {
+            const auto diff = std::fabs(constant_value - value);
+            if ((exact_comparison && (std::fabs(constant_value - value) > std::numeric_limits<float>::epsilon())) ||
+                (diff > max_diff)) {
+                return false;
+            }
+
+            need_to_check_intervals = true;
+        }
+    }
+
+    if (need_to_check_intervals) {
+        auto tmp_fq = as_type_ptr<opset1::FakeQuantize>(fq->clone_with_new_inputs({
+            constant,
+            fq->get_input_node_shared_ptr(1),
+            fq->get_input_node_shared_ptr(2),
+            fq->get_input_node_shared_ptr(3),
+            fq->get_input_node_shared_ptr(4)}));
+        auto result = NetworkHelper::fold_fake_quantize(tmp_fq, false, 1);
```

Comment:
These changes contain a conflict with #17096. Let's discuss offline in which order these PRs will be merged